core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerClientProvider.scala

```diff
@@ -32,7 +32,7 @@ class HttpBillingProfileManagerClientProvider(baseBpmUrl: Option[String]) extend
     val client: ApiClient = new ApiClient()
 
     val clientConfig = new ClientConfig()
-    clientConfig.connectorProvider(new JdkConnectorProvider)
```

Comment:
I suggest a comment here explaining the nature of the workaround for future devs.